run_benchmarks.rb

```diff
@@ -347,6 +347,16 @@ def run_benchmarks(ruby:, ruby_description:, categories:, name_filters:, out_pat
     args.harness = v
   end
 
+  opts.on("--warmup=N", "the number of warmup iterations (default: 15)") do |n|
+    ENV["WARMUP_ITRS"] = n
+  end
+
+  opts.on("--bench=N", "the number of benchmark iterations (default: 10)") do |n|
+    ENV["WARMUP_ITRS"] ||= "0"
```

Comment:
Seems a bit odd to default WARMUP_ITRS to 0, but only if --bench is specified.

Reply:
You *do* document it that way in the README, so I assume this is on purpose. And that very often matches my usage pattern. I'm just not sure if this will be expected behaviour.

Reply:
Okay, I dropped that feature to avoid confusion https://github.com/Shopify/yjit-bench/pull/261/commits/9ddaeae8e0c1a4806dc440eee43c91943d1f73af.